Affiliated-Project-Policy.md

```diff
@@ -0,0 +1,163 @@
+
+Among the projects with an existing OBF affiliation status, representative examples are BioPerl and Biopython.
+
+### Affiliated OBF Project
+
+Affiliated OBF projects are active bioinformatics open-source initiatives with an
+active developer community, but a limited or small user base. Such projects may have a narrow focus, or be in the early phases of adoption. Even if small by comparison to Core OBF Projects, their developer and user communities will nevertheless be very active, and their focus will be highly relevant to the OBF. These projects will also be active members of
+the OBF organization.
+
+#### Requirements for Affiliated Project status
+To be an affiliated project, a project must show the following:
+
+- An active community with defined leadership. Evidence of this could include recent pull requests being reviewed / accepted, blog posts, or community events/conferences.
+- The project must have an Open Source licence, either [OSI-approved](https://opensource.org/licenses) or one that was agreed upon by the OBF Board as compatible.
+- The project must have a Code of Conduct. Linking directly to the OBF Code of Conduct is acceptable.
+
+### Candidate OBF Project
+
+Candidate OBF Projects have the aspiration and promise to
+develop themselves into an Affiliated OBF Project. They may already meet most or all criteria for Affiliated Project status except for having been active members of the OBF organization.
+
+This status presents the path for projects previously unaffiliated with the OBF to attain official affiliation. Candidate Projects will be in this status for only a limited amount of time, typically one year, and no more than 3 years in a row.
+
+Candidate projects must fulfil all criteria for the Affiliated OBF Project status before they can officially become affiliated.
+
+In order to apply for Candidate status, [create a GitHub issue](https://github.com/OBF/obf-docs/issues/new?labels=candidate-project-application&template=candidate-project-application.md) with the following information:
+
+- URL of the project
+- Contact information for the project maintainers
+- Proof of compliance with Affiliate Status requirements where possible, e.g. licence, code of conduct, etc. or plan to implement any missing requirements.
+
+### Other projects
+
+Projects other than those under the stated affiliation categories may
+have a de-facto relationship with OBF or one of its official member
+projects, but they do not have a recognized affiliation status. As
+such, their status is not voted on.
+
+These projects may include student projects with OBF community
+participation, including projects executed under the OBF umbrella,
+and other projects considered active and interesting to the wider
+bioinformatics community. They may be listed and described on the
+OBF wiki so long as it is evident that they do not have an official
```

Comment:
obf website, not obf wiki

Reply:
Fixed in 8f483366b916fb9ea63d2268036299f815bf140d